src/components/Sidebar.jsx

```diff
@@ -0,0 +1,195 @@
+        <ul className="uppercase">
+          <li className="px-8 py-4 transition-colors cursor-pointer hover:text-green-400">
+            <AiOutlineCloseCircle onClick={toggleMenu} className="text-3xl" />
+          </li>
+          { userInfo.role === 'admin' ? (
+            <li className="px-8 py-4 transition-colors cursor-pointer hover:text-green-400">
+              <p className="text-center font-bold">You are an admin</p>
+            </li>
+          ) : (<></>)}
+          <li className="px-8 py-4 transition-colors border-b cursor-pointer border-white-700 hover:text-green-400">
+            <button
+              type="button"
+              onClick={() => directLink('/doctors')}
+              className="block w-full"
+            >
+              List of Doctors
+            </button>
+          </li>
+          <li className="px-8 py-4 transition-colors border-b cursor-pointer border-white-700 hover:text-green-400">
+            <button
+              type="button"
+              onClick={() => directLink('/appointment-list')}
+              className="block w-full"
+            >
+              Check your Appointments
+            </button>
+          </li>
+          <li className="px-8 py-4 transition-colors border-b cursor-pointer border-white-700 hover:text-green-400">
+            <button
+              type="button"
+              onClick={() => directLink('/create-appointment')}
+              className="block w-full"
+            >
+              Schedule a new Appointment
+            </button>
+          </li>
+          { userInfo.role === 'admin' ? (
+            <>
+              <li className="px-8 py-4 transition-colors border-b cursor-pointer border-white-700 hover:text-green-400">
+                <button
+                  type="button"
+                  onClick={() => directLink('/add-docs')}
+                  className="block w-full"
+                >
+                  Add a new Doctor
+                </button>
+              </li>
+              <li className="px-8 py-4 transition-colors border-b cursor-pointer border-white-700 hover:text-green-400">
+                <button
+                  type="button"
+                  onClick={() => directLink('/delete-docs')}
+                  className="block w-full"
+                >
+                  Delete a Doctor
+                </button>
+              </li>
+            </>
+          ) : (<></>)}
+          <li className="px-8 py-4 transition-colors border-b cursor-pointer border-white-700 hover:text-green-400">
+            <button
+              type="button"
+              onClick={() => directLink('/logout')}
+              className="block w-full"
+            >
+              Sign out
+            </button>
+          </li>
```

Comment:
These button list items could be abstracted into a subcomponent and you can clean up the code by using some sort of array/map combination. I strongly recommend this.